.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Run watir-rails CI
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.ruby-version == 'debug' }}
+    strategy:
+      matrix:
+        BUNDLE_GEMFILE: [gemfiles/Gemfile.rails-3.x, gemfiles/Gemfile.rails-4.x, Gemfile]
+        ruby-version: [2.6.3, debug]
+        exclude:
```

Comment:
Please explain this whole `exclude` block - currently I'm reading that `matrix.BUNDLE_GEMFILE` is the same as `matrix.exclude`, which makes it feel that everything that is included is also excluded.

Reply:
We're setting hashes that are skipped if matches the expanded value of the matrix, so right now we have
```
- { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-3.x, ruby-version: debug }
- { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-3.x, ruby-version: 2.6.3 }
- { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-4.x, ruby-version: debug }
- { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-4.x, ruby-version: 2.6.3 }
- { BUNDLE_GEMFILE: Gemfile, ruby-version: debug }
- { BUNDLE_GEMFILE: Gemfile, ruby-version: 2.6.3 }
```
after expansion and only 3.x & 4.x with debug is skipped (we're running only `Gemfile` on `debug`)

Hope this helps, it's documented [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).